### DISCUSSION TITLE
browser-refreshing-sharepoint-page/

### DISCUSSION THREAD

**Discussion by giscus** [estruyf/eliostruyf-comments#621] [Comments]
# browser-refreshing-sharepoint-page/

Learn how to fix your SharePoint page from going into an endless refreshing loop when third-party cookies are blocked.

https://www.eliostruyf.com/browser-refreshing-sharepoint-page/

**Comment by y0u-s** [comment]
Hello, thank you for addressing the current issue our organization is encountering. In Solution One, you mentioned the need to approve a specific permission scope in the SharePoint Admin Center. Could you please specify the exact name of the permission scope that requires approval?

Additionally, if there's only one pending request visible in the Admin Center, but it doesn't seem related to the mentioned permission scope, does Solution One still apply in this scenario?

Thank you in advance for clarifying.

**Reply by estruyf** [reply]
The permission scope depends on what your solution needs. Normally, this is defined in the `webApiPermissionRequests` property of the `config/package-solution.json` file. 

If not defined, it has to be verified in the code which permission scope it is and then add it to the `webApiPermissionRequests`.

An alternative way is to validate it in Chrome or by allowing the third-party cookies in your browser and validating the token retrieval call to `https://login.microsoftonline.com`. On that call, you should see the requested `scopes`. For example: `&scope=https%3A%2F%2Fgraph.microsoft.com%2F.default%20openid%20profile%20offline_access`.
